app/src/pages/project/SpanFilterConditionField.tsx

```diff
@@ -168,7 +174,13 @@ function filterConditionCompletions(context: CompletionContext) {
       {
         label: "Hallucinations",
         type: "text",
-        apply: "evals['Hallucination'].label == 'hallucinated'",
+        apply: "annotations['Hallucination'].label == 'hallucinated'",
+        detail: "macro",
+      },
+      {
+        label: "Annotations",
+        type: "text",
+        apply: "annotations['Hallucination'].label == 'hallucinated'",
```

Comment:
tried for a long time (probably too long) to get a macro with that would follow:
"annotations['${1}'].label == '${2}'"

and start the cursor at ${1} and go to ${2} on tab with no luck so just copied eval

Reply:
damn, I think it's worth investing here a bit. We also have some short-term fixes we can try